pkg/pool-linear/contracts/LinearPoolRebalancer.sol

```diff
@@ -24,6 +24,8 @@ import "@balancer-labs/v2-interfaces/contracts/pool-linear/ILinearPool.sol";
 import "@balancer-labs/v2-solidity-utils/contracts/math/FixedPoint.sol";
 import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/SafeERC20.sol";
 
+import "@balancer-labs/v2-interfaces/contracts/solidity-utils/openzeppelin/IERC20.sol";
```

Comment:
@rabmarut Somehow this slipped my eyes. Will need to look into this dif more.

Reply:
Yup, all of these changes should be reverted. In general, we don't want to make changes to core files when adding new integrations. If we notice that a change is really required, we can communicate that to BLabs.

Reply:
I updated the branch: see additional changes here: https://github.com/mkflow27/balancer-v2-monorepo/pull/2/files

LinearPoolRebalancer changes reverted.